extensions/positron-python/src/test/common/platform/utils.ts

```diff
@@ -28,7 +28,11 @@ export const SUPPORTS_SYMLINKS = (() => {
     return true;
 })();
 export const SUPPORTS_SOCKETS = (() => {
-    if (WINDOWS) {
+    // --- Start Positron ---
+    // Upstream doesn't run on OSX but that seems to fail too.
+    // if (WINDOWS) {
+    if (WINDOWS || OSX) {
+        // --- End Positron ---
```

Comment:
This was really weird to track down. After the upstream changes in https://github.com/microsoft/vscode-python/pull/25404, some file system tests started failing on MacOS, which upstream does not run on. But they pass on ubuntu.

I noticed all the failing tests were gated by `if (!SUPPORTS_SOCKETS) {skip()}`, so I tried this, and it makes everything pass. I don't know if this is something we should worry about. Probably not, since they decided they can skip all these tests on Windows anyway.

Reply:
Agreed. I vaguely recall that we have skipped tests on macos for this reason before